src/Api/Collection.php

```diff
@@ -19,12 +19,39 @@
  */
 class Collection
 {
+    /**
+     * @var Api
```

Comment:
This should probably be \Kirby\Api\Api, right? There are a few more here, which probably need to have absolute namespaces. I.e. \Exception. Etc.